src/templates/finops-hub/modules/storage.bicep

```diff
@@ -160,6 +160,18 @@ resource dfsPrivateDnsZone 'Microsoft.Network/privateDnsZones@2024-06-01' = {
   properties: {}
 }
 
+resource queuePrivateDnsZone 'Microsoft.Network/privateDnsZones@2024-06-01' = {
+  name: 'privatelink.queue.${environment().suffixes.storage}'
+  location: 'global'
+  properties: {}
+}
+
+resource tablePrivateDnsZone 'Microsoft.Network/privateDnsZones@2024-06-01' = {
+  name: 'privatelink.table.${environment().suffixes.storage}'
+  location: 'global'
+  properties: {}
+}
+
```

Comment:
Do we need these? Shouldn't we leave them out so there's no access to features we don't use?
```suggestion
```

Reply:
We need these DNS zones for ADX support.